test/libsolidity/util/TestFunctionCall.h

```diff
@@ -42,33 +42,42 @@ namespace solidity::frontend::test
 class TestFunctionCall
 {
 public:
-	TestFunctionCall(FunctionCall _call): m_call(std::move(_call)) {}
+	enum class RenderMode
+	{
+		Expectations,
+		ResultsOnly,
+		GasResults
+	};
```

Comment:
How about these? Now very succint but it took me some time before I fully realized what the current values actually mean.
```suggestion
	enum class RenderMode
	{
		ExpectedValuesExpectedGas,
		ActualValuesExpectedGas,
		ExpectedValuesActualGas,
	};
```

Alternatively, you could reduce it to two values (`TestExpectations` and `TestResults`) and instead have two separate parameters, one for gas and one for other values.